sdk/eventhub/Azure.Messaging.EventHubs/tests/Diagnostics/DiagnosticsTests.cs

```diff
@@ -295,21 +295,30 @@ public async Task PartitionPumpCreatesScopeForEventProcessing()
                         });
                 });
 
-            var clientMock = new Mock<EventHubConnection>();
-            clientMock.Setup(c => c.CreateTransportConsumer("cg", "pid", It.IsAny<EventPosition>(), It.IsAny<EventHubConsumerClientOptions>())).Returns(consumerMock.Object);
+            var connectionMock = new Mock<EventHubConnection>();
```

Comment:
Thanks.   I have a feeling we'll be cleaning this kind of thing up for a while.   I hate wide scope renames.